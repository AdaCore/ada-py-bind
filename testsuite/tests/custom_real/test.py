import gen

p = gen.Shape()
p.position.x = 0.52
p.position.y = -0.52

print p.position.x
print p.position.y

try:
    p.position.x = 1.2
except Exception as e:
    print e.message

print p.position.x
