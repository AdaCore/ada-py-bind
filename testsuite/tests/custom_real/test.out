0.52
-0.52
Native Ada exception: 
py_bind-types.adb:192 range check failed
Traceback: 
0.52
